File: server/Models/Logo.js
const mongoose = require('mongoose')

const LogoSchema = mongoose.Schema({

    nameth:{
        type: String
    },
    nameen:{
        type: String
    },
    icon:{
        type: String
    },
    path:{
        type: String
    }

})

module.exports = mongoose.model('logo',LogoSchema)

File: Go_api/models/logo.go
package models

import "go.mongodb.org/mongo-driver/bson/primitive"

type Logo struct {
	ID primitive.ObjectID `bson:"_id,omitempty" json:"-"`
	Nameth  string `bson:"nameth" json:"nameth"`
	Nameen string `bson:"nameen" json:"nameen"`
	Icon string `bson:"icon" json:"icon"`
	Path string `bson:"path" json:"path"`
}